justfile

```diff
@@ -178,3 +178,7 @@ publish-linux-target TARGET=DEFAULT_TARGET *ARGS='':
 # Generate changelog for a release
 generate-changelog *ARGS:
     ./ci/changelog/changelog.sh {{ARGS}}
+
+# Remove unused/unreferenced images
+clean-unused-images:
+    ./ci/admin/check_unused_images.sh
```

Comment:
Although on a second look, would probably be good to also pass *ARGS here

Reply:
ah right...I forgot to add the default `--delete` option as well.

Reply:
I've also squashed the fixups now (as it was only a small edit)